### PR TITLE
fixed overrides of operations in internal ops table

### PR DESCRIPTION
**Pull request by chaudum**
by using unique ids that are based on operation id and job id

### CODE REVIEW

File: CHANGES.txt [hunk 1]
```diff
@@ -5,6 +5,10 @@ Changes for Crate
 Unreleased
 ==========
 
+ - Fixed issue that caused certain operations not showing up in operations_log
+   table and certain operations showing up in operations_log table before the
```
Comment (seut) [inline CHANGES.txt]: hm bit long and hard to understand, suggestion:
Fixed issue that caused certain operations not showing up at all in operations_log while other marked as finished even before they actually finished

**Comment by seut** [comment]
otherwise ack

File: CHANGES.txt (hunk 1 above)
Comment (chaudum) [inline CHANGES.txt]: yes, your sentence is easier to understand!

**Comment by seut** [comment]
ack

**Pull request merged**